tools/devops/automation/templates/common/install-qa-provisioning-profiles.yml

```diff
@@ -0,0 +1,66 @@
+parameters:
+- name: env
+  type: object
+
+- name: secureFiles
```

Comment:
It's finally all green! There's just one question that remains for me: how do I update the existing certificates/profiles and/or add new ones? In other words, I would love to get the CREATE.md/EXPIRED.md/README.md documents here: https://github.com/xamarin/maccore/tree/main/tools/provisioning-profiles moved/ported here as well.

Reply:
Mmm probably they need to go into the private submissions test repo and have the readme here point to those instead

Reply:
Not quite:
![image](https://github.com/user-attachments/assets/95915b0d-adc8-4bab-85c4-34a3127c2f16)

Yes, I will update the documentation in a seperate PR.